src/applications/claims-status/containers/YourClaimsPageV2.jsx

```diff
@@ -68,6 +68,7 @@ class YourClaimsPageV2 extends React.Component {
     if (!this.props.loading && prevProps.loading) {
       setPageFocus();
     }
+    setPageFocus();
```

Comment:
Can this be reduced to just `setPageFocus();`, without the condition on line 68?

Reply:
Would this conditional logic prevent a case where the focus was being reset outside of a loading situation? I'm thinking specifically about the `Find out why we sometimes combine claims` link. It opens a modal window, and should re-take focus when the modal window is closed. Maybe the two are unrelated?

Reply:
Okay, so that's a separate issue. I will see if it has been logged previously and if not, will add a new issue.